docs-www/src/pages/landing.js

```diff
@@ -7,7 +7,9 @@ const LandingPage = () => (
   <div className={styles.landing}>
     <div>
       <h1 className={styles.landing__header}>React-carousel</h1>
-      <p className={styles.landing__subheader}>Feature-rich, react-way react component that does not suck</p>
+      <p className={styles.landing__subheader}>
+        Feature-rich, react-way react component that does not suck
```

Comment:
```suggestion
        🎠 A pure React carousel, powered by Brainhub (craftsmen who ❤️ JS) and IssueHunt, open for new feature proposals
```

Reply:
and the same in `docs-www/gatsby-config.js`

Reply:
WDY about   ```A pure extendable React carousel, powered by <a href="https://brainhub.eu/">Brainhub</a> (craftsmen who ❤️ JS) ```?

Reply:
We can decide about that together in the Open Source team.

IMO:
- `extendable` -> definitely yes
- skipping `IssueHunt` -> I'm not sure; the advantages would be making it shorter and focusing only on Brainhub but on the other hand, telling about IssueHunt can attract external contributors

Reply:
ok, let's discuss it tommorow